### DISCUSSION TITLE
sqlite3 built in function concat is not recognized.

### DISCUSSION THREAD

**Discussion by Bob3056** [pawelsalawa/sqlitestudio#5423] [General]
### Details
[19:23:33] Error while executing SQL query on database 'Test': no such function: concat

### Steps to reproduce
SELECT concat( "A", "B" )

### Operating system
Windows 10

### SQLiteStudio version
3.44

**Comment by pawelsalawa** [comment]
SQLiteStudio version is 3.4.4, not 3.44. It comes with SQLite version 3.34.1, while the function you mention `concat()` was introduced in SQLite 3.44.

**Comment by pawelsalawa** [comment]
You can update SQLite version in your SQLiteStudio installation directory - just download fresh SQLite3 dll file from sqlite.org and replace the one in SQLiteStudio's installation directory.
